helm/olmv1/templates/cert-manager/certificate-olmv1-system-operator-controller-cert.yml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.options.certManager.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  annotations:
+    {{- include "olmv1.annotations" . | nindent 4 }}
+  labels:
+    app.kubernetes.io/name: {{ include "olmv1.label.name" . }}
+    {{- include "olmv1.labels" . | nindent 4 }}
+  name: operator-controller-cert
+  namespace: {{ .Values.namespaces.olmv1.name }}
+spec:
+  dnsNames:
+    - operator-controller-service.{{ .Values.namespaces.olmv1.name }}.svc
+    - operator-controller-service.{{ .Values.namespaces.olmv1.name }}.svc.cluster.local
+  issuerRef:
+    group: cert-manager.io
+    kind: ClusterIssuer
+    name: olmv1-ca
+  privateKey:
+    algorithm: ECDSA
+    rotationPolicy: Always
+    size: 256
+  secretName: operator-controller-cert
```

Comment:
@tmshort we changed the name, it was before olmv1-cert 
But I think it is fine, downstream would we need to do something when upgrade to delete the old secret?